includes/class-windows-azure-replace-media.php

```diff
@@ -55,13 +64,21 @@ public function __construct() {
 		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_replace_media_script' ) );
 
 		// ajax event to replace media
-		add_action( 'wp_ajax_nopriv_azure-storage-media-replace', array(  $this, 'process_media_replacement' ) );
-		add_action( 'wp_ajax_azure-storage-media-replace', array(  $this, 'process_media_replacement' ) );
-
-		// Ajax event to set transient for replacement
-		add_action( 'wp_ajax_nopriv_azure-storage-media-replace-set-transient', array(  $this, 'set_media_replacement_transient' ) );
-		add_action( 'wp_ajax_azure-storage-media-replace-set-transient', array(  $this, 'set_media_replacement_transient' ) );
+		add_action( 'wp_ajax_nopriv_azure-storage-media-replace', array( $this, 'process_media_replacement' ) );
+		add_action( 'wp_ajax_azure-storage-media-replace', array( $this, 'process_media_replacement' ) );
 
+		// Set allowed mime types to be replaced
+		$this->allowed_types = apply_filters(
```

Comment:
Not sure if we do this consistently in this plugin but would be great to add proper hook documentation here: https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#4-hooks-actions-and-filters